bank_statement_parse_nl_ing_mt940/mt940.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+"""Implement BankStatementParser for MT940 IBAN ING files."""
+##############################################################################
+#
+#    Copyright (C) 2014-2015 Therp BV <http://therp.nl>.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+import re
+from openerp.addons.bank_statement_parse_mt940.mt940 import (
+    MT940, str2amount, get_subfields, handle_common_subfields)
+
+
+class MT940Parser(MT940):
+    """Parser for ing MT940 bank statement import files."""
+
+    tag_61_regex = re.compile(
+        r'^(?P<date>\d{6})(?P<line_date>\d{0,4})'
+        r'(?P<sign>[CD])(?P<amount>\d+,\d{2})N(?P<type>.{3})'
+        r'(?P<reference>\w{1,50})'
+    )
+
+    def __init__(self):
+        """Initialize parser - override at least header_regex."""
+        super(MT940Parser, self).__init__()
+        self.mt940_type = 'ING'
+
+    def handle_tag_61(self, data):
+        """get transaction values"""
+        super(MT940Parser, self).handle_tag_61(data)
+        re_61 = self.tag_61_regex.match(data)
+        if not re_61:
+            raise ValueError("Cannot parse %s" % data)
+        parsed_data = re_61.groupdict()
+        self.current_transaction.transferred_amount = (
```

Comment:
Refactor `transferred_amount` to `amount` (and please validate the amount of a transaction in the test, because it didn't get caught). Without the change, imported statements contain no amounts.